addons/mail/static/src/components/composer/composer.scss

```diff
@@ -40,6 +40,11 @@
     &.o-composer-is-compact {
         max-height: 100px;
     }
+
+    &:not(.o-composer-is-compact) {
+        overflow-y: auto;
+        max-height: 300px;
```

Comment:
I think this is too big, it doesn't leave enough space at all for the message list.

I think we might need a "vertical size responsive value" here.

In fact on 720px height screen (but large enough to not go to chatter below), there is a triple scroll bar madness because the "send" button is actually too low so the whole page gets a scroll because of the chatter, and it even "breaks" the left border at the bottom because `o_Chatter` becomes smaller than it's child `o_Chatter_fixedPanel`.

Reply:
I have no idea what is the best solution and while there is improvement our current approach does not look ideal yet.

Maybe having a fixed area was a mistake?

Reply:
I have no idea, I'm not really a CSS purist ..

Reply:
I think it's fine for now.

TBH, I suspect we will have some component concerns in modeling in the future, and sizing of composer should be explicit there when needed. To be a bit more precise, I see the modeling putting some size constraints on container of composer view, not the composer view itself (because generally auto-growing/shrinking items have limits that are dictates by containers). Definitely out-of-scope of PR, and for now I'm just dropping some rough ideas...

Reply:
so :) ?

Reply:
I guess good enough for now. But we can be sure this is not the last time we hear about this issue.